src/include/common/types/types.h

```diff
@@ -426,6 +426,7 @@ class LogicalTypeUtils {
     static bool isNested(const LogicalType& dataType);
     static std::vector<LogicalType> getAllValidComparableLogicalTypes();
     static std::vector<LogicalTypeID> getNumericalLogicalTypeIDs();
+    static std::vector<LogicalType> getIntegerLogicalTypes();
```

Comment:
I think you only need to collect LogicalTypeID for integer types. Let's try to be consistent with numerical. Also double check if range(2.1, 3.1) is defined or not?

Reply:
range(2.1, 3.1) is not defined.
Both Python and Neo4j only supports integer types.